### ISSUE TITLE
check on taxonomic source for Poa nascopieana

### ISSUE THREAD

**Issue by manions** [Canadensys/vascan-data#235]
[Originally posted on GoogleCode (id 2373) on 2015-01-14]

<b>(This is the template to report a data issue for Vascan. If you want to</b>
<b>report another issue, please change the template above.)</b>

<b>What is the URL of the page where the problem occurs?</b>
http://data.canadensys.net/vascan/taxon/7885?lang=en

<b>What data are incorrect or missing?</b>
Taxonomic source is FNA 24 2007 but can't find in this source

<b>What data are you expecting instead?</b>
Listed with Porsild 1957 but any other source?
Just wondering also what you think of this taxon please, a good taxon? 
Thanks

<b>If applicable, please provide an authoritative source.</b>

**Comment by brouille** [comment]
FNA 24 had it as a name of uncertain application
Here is a note in Tropicos concerning this taxon: 
HT: N. Polunin 549; 3-5 Sep 1934; Canada: Baffin Island: Pangnirtung (BM-528099; IT: CAN) Fruit replaced by sclerotia, florets w/ a distinct tuft of hairs (web), lemmas short pub 5N and a bit btwn, ligules < 1.3mm, extravaginal branching clearly apparent, lower sheaths with short stiff retrorse hairs near throat and base, blade folded smooth as in arctica or pratensis with margins inrolled, aprsely scabrous adaxially, palea keels pilose medially, w/ pilose hairs between, panicle smooth with bracts at 2 nodes, possibly it is diseased P. arctica; RJS 2010 
and
Note: a diseased and deformed plant [fide: Soreng] 
and in NWG :    Doubtful or dubious
It is interesting that Aiken et al. 2007 had the taxon, but not the current online version of the Fl. Artic Archipelago online.
In conclusion, I will place it in synonymy, though with reservation, with P. arctica
I should not have kept it. It may have been in an early draft of FNA Poaceae.
Luc
